jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaLayerConfigurations.java

```diff
@@ -42,12 +44,12 @@
     EXTRA_FILES("extra files", "/");
 
     private final String label;
-    private final String extractionPath;
+    private final Path extractionPath;
 
     /** Initializes with a label for the layer and the layer files' default extraction path root. */
     LayerType(String label, String extractionPath) {
       this.label = label;
-      this.extractionPath = extractionPath;
+      this.extractionPath = Paths.get(extractionPath);
```

Comment:
It's mostly since this is just a constructor for the enum and pushing it to the caller would require putting `Paths.get` multiple times in the enum list above.